### ISSUE TITLE
Tables/Strip windows enlarging on layout with scales slightly larger than 1

### ISSUE THREAD

**Issue by aleokdev** [emilk/egui#1568]
**Describe the bug**
The table/strip windows in the demo app grow slightly larger with each frame rendered when using scales *sligtly* larger than 1. Tested with 1.08. [A video describes it well.](https://user-images.githubusercontent.com/16354904/166656556-22899177-3969-45f8-a600-38d87f3cb20a.mp4)

**To Reproduce**
1. Open the demo app.
2. Open the strip & table demos.
3. Set the scale to 1.08.
4. Move the cursor around or set the repaint mode to continuous.
Also happens on web (Just set the scale by using Ctrl+scroll wheel or Ctrl + +/-)

**Expected behavior**
The strip and table windows should keep their original size.

**Additional context**
Tested on latest master + web demo

**Comment by DusterTheFirst** [comment]
I can reproduce this on the latest web demo aswell: 

https://user-images.githubusercontent.com/14093962/167803740-fd00cd72-bc70-4130-9c4c-ab648914f492.mp4

**Comment by emilk** [comment]
Duplicate of https://github.com/emilk/egui/issues/1543 ?

**Comment by aleokdev** [comment]
> Duplicate of https://github.com/emilk/egui/issues/1543 ?

Yup, didn't see that one! Looks like the cause of the problem is scaling, and it doesn't just happen with tables though.

**Comment by peragwin** [comment]
Any updates, potential workarounds, or suggestions where to look if I wanted to take a stab at this?

**Comment by Vaelio** [comment]
Hi,

I'm not sure if its related but I'm experiencing something similar with tables:

I have a table in a window inside centerpanel and somehow when i resize the main window , for example by making it fullscreen, the table width grows with it (i can see it by looking at the scroll bar).
I'm fairly confident it is linked to the Table widget because i have other widgets in their own windows and there's no problem.

Here's the size by default (the scrollbar is already there even though the widget fits in width):
![image](https://user-images.githubusercontent.com/6543163/186887925-6256073f-fa36-462f-88b1-12d192a79b37.png)

Here's the size when i increase the main window size:
![image](https://user-images.githubusercontent.com/6543163/186888038-51b92b2b-9136-4c9a-872e-69e07802733a.png)
